test/fixtures/es-modules/print_3.mjs

```diff
@@ -0,0 +1 @@
+console.log(3);
```

Comment:
This file should be named `print-3.mjs` to follow the pattern of the other files in this folder.